build.gradle

```diff
@@ -12,22 +12,32 @@ buildscript {
 
 plugins {
     id 'com.palantir.git-version' version '0.4.0'
-    id 'org.inferred.processors' version '1.1.5'
 }
 
-apply plugin: 'java'
 apply plugin: 'com.palantir.baseline-config'
 apply plugin: 'com.palantir.baseline-idea'
-apply plugin: 'com.palantir.baseline-checkstyle'
-apply plugin: 'com.palantir.baseline-findbugs'
-apply plugin: 'com.palantir.jacoco-coverage'
-apply from: "${rootDir}/gradle/publish.gradle"
 
 repositories {
```

Comment:
allprojects are for both root project and subprojects. findbugs/checkstyle/eclipse/jacoco and publishing are not needed for the root